vmware-tanzu/nsx-operator
# Remove Subnet/SubnetSet Finalizer

This PR tries to fix the issue that Subnet CR deletion takes about 2 minutes (due to the NSX IP release issue), resulting in a prolonged Namespace deletion time.
Remove NetworkInfo Finalizer in https://github.com/vmware-tanzu/nsx-operator/pull/778 

TestDone:
1. Create a Subnet and check its status; update the Subnet and check its status; delete the Subnet and check for any stale Subnet in NSX.(included in e2e test cases)
2. Create a SubnetSet and check its status; update the SubnetSet and check its status; delete the SubnetSet and check for any stale Subnet in NSX.(included in e2e test cases)
3. Rebase #778 and test the deletion of a Namespace, then delete the Pod, the Namespace will not stuck in terminating status.
